silver/tests/commands/test_generate_docs.py

```diff
@@ -1445,7 +1447,7 @@ def test_canceled_subscription_with_trial_and_trial_underflow(self):
         )
 
         subscription.cancel(when=Subscription.CANCEL_OPTIONS.END_OF_BILLING_CYCLE)
-        subscription.cancel_date = dt.date(2015, 02, 28)
+        subscription.cancel_date = dt.date(2015, 0o2, 28)
```

Comment:
These should not be octals, as in `0o2` should simply be `2`. I will change them if you can't be bothered.

Reply:
Agh. That script I ran did this. I ran a quick search and found a couple more dates with 0o. It doesn't appear to cause any harm when creating the actual dates. Should be fixed now.